pkgs/development/web/rover/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, perl
+, openssl
+, pkg-config
+, darwin
+, stdenv
+}:
+
+let
+  # The build fetches this dynamically from https://graphql.api.apollographql.com/api/schema
+  apolloSchema = builtins.readFile ./schema.graphql;
+  # The build fetches this from the headers of https://graphql.api.apollographql.com/api/schema
+  etagId = builtins.readFile ./etag.id;
+in
+rustPlatform.buildRustPackage rec {
+  pname = "rover";
+  version = "0.1.10";
+
+  src = fetchFromGitHub {
+    owner = "apollographql";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0vmm35b9d7xw160pn797z78vj8jaav0qb6dw7n4ld8ssj076dsrh";
+  };
+
+  nativeBuildInputs = [ perl pkg-config ];
+  buildInputs = [ openssl ] ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```

Comment:
```suggestion
  buildInputs = [ openssl ] ++ lib.optional stdenv.isDarwin Security;
```

Reply:
I don't think `Security` is imported at this point, so this suggestion fails with `undefined variable Security`

Reply:
You need the following in the top-level entry.
```
inherit (darwin.apple_sdk.frameworks) Security;
```

Reply:
okay if that's the convention :+1: done in babbdbb2859be8949249cd17c22168a3b581c7a3